<launch>
    <arg name="map_size_x" default="200.0"/>
    <arg name="map_size_y" default="200.0"/>
    <arg name="map_size_z" default=" 5.0"/>

    <arg name="init_x" default="-20.0"/>
    <arg name="init_y" default="-20.0"/>
    <arg name="init_z" default="  0.5"/>

    <node pkg ="bezier_traj" name ="random_obs_generation_node" type ="random_obs_generation_node" output = "screen">
        <!--remap from="~odometry"   to="/odom/fake_odom"/-->
        <param name="init_state_x"   value="$(arg init_x)"/>
        <param name="init_state_y"   value="$(arg init_y)"/>
        <param name="map/x_size"     value="$(arg map_size_x)" />
        <param name="map/y_size"     value="$(arg map_size_y)" />
        <param name="map/z_size"     value="$(arg map_size_z)" />
        <param name="map/obs_num"    value="8320"/>
        <param name="map/resolution" value="0.5"/>
        <param name="ObstacleShape/lower_rad" value="0.3"/>
        <param name="ObstacleShape/upper_rad" value="1.6"/>
        <param name="ObstacleShape/lower_hei" value="1.0"/>
        <param name="ObstacleShape/upper_hei" value="6.0"/>
        <param name="sensing/radius" value="15.0"/>
        <param name="sensing/rate"   value="10.0"/>
    </node>

    <node pkg="bezier_traj" name="bezier_traj_node" type="bezier_traj_node" output="screen">
    	<remap from="/map" to="/random_forest" />
    </node>

    <node pkg="bezier_traj" name="odom_generation" type="odom_generation" output="screen" />

    <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find bezier_traj)/launch/simulation.rviz" />
</launch>
